modules/ROOT/pages/design-ghs-install-authorize.adoc

```diff
@@ -0,0 +1,73 @@
+
+=== Before You Begin
+
+To install the GitHub Synchronization application as a GitHub organization administrator, you must have:
+
+* A GitHub account
+* A membership in the GitHub organization where the application will be installed
+* The administrator permission to access the GitHub organization (as configured through your GitHub account settings)
+
+=== Install the GitHub Synchronization application
+
+. Navigate to the https://github.com/apps/mulesoft-github-integration[GitHub Synchronization application page^].
+. Click *Install*.
+. In the *Install MuleSoft GitHub Integration* dialog, select an organization in which to install the application.
+. Click *All repositories* to make all repositories available, or click *Only select repositories* and specify individual repositories.
+. Click *Install*.
+
+The application must be authorized in API Designer to complete the integration with GitHub. See xref:design-ghs-install-authorize.adoc#authorize-the-github-sync-application-in-api-designer[Authorize the GitHub Sync Application in API Designer].
+
+== Install the GitHub Synchronization Application as a GitHub User
+
+Installing the GitHub Synchronization application in a GitHub user account enables you to sync your individual GitHub user repositories. 
+
+=== Before You Begin
+
+To install the GitHub Synchronization application as a GitHub user, you must have:
+
+* A GitHub account.
+* A membership in the GitHub organization where the application will be installed.
+
+=== Install the GitHub Synchronization Application
+
+. Navigate to the https://github.com/apps/mulesoft-github-integration[GitHub Synchronization application page^].
+. Click *Install*.
+. In the *Install MuleSoft GitHub Integration* dialog, select a GitHub user account (typically the first items in the displayed list).
+. Click *All repositories* to make all repositories available, or click *Only select repositories* and specify individual repositories.
+. Click *Install*.
+
+The application must be authorized in API Designer to complete the integration with GitHub. See xref:design-ghs-install-authorize.adoc#authorize-the-github-sync-application-in-api-designer[Authorize the GitHub Synchronization Application in API Designer].
+
+== Authorize the GitHub Synchronization Application in API Designer
+
+To authorize the GitHub Synchronization application ensure your user ID is assigned the Design Center Developer permission.
+
+. Sign in to GitHub.
+. On the Design Center *Projects* page, click *Create new*.
+. Select *New API Spec*.
+. In the *New API Spec* panel, click *Authorize the MuleSoft app now to get started*.
+. Click *Authorize*.
+. In the subsequent dialog, click *Authorize MuleSoft GitHub Integration*.
+
+You are redirected to API Designer, which displays an *Authorized Successfully* message. You can now complete the process that originally triggered the application authorization.
+
+== Troubleshooting GitHub Synchronization Application Installation
```

Comment:
The subsections are a bit confusing because the first title **Deauthorize the GitHub...** contains some troubleshooting steps, but the next title: **The *GitHub Organization* list is empty** seems to be an error rather than steps, I'd suggest reorganizing this section by removing the titles and leaving the content, something like: 

== Troubleshooting GitHub Synchronization Application Installation

If you encounter errors when attempting to authorize the GitHub Synchronization application, try deauthorizing and then reauthorizing the GitHub Synchronization application: 

1. Click *Revoke* in... 
2. ...

If you see an empty list when... , ensure that...

Reply:
Oh good call, thanks